web/samples/uncompiled.html

```diff
@@ -23,15 +23,7 @@
     </style> 
  
     <!-- Insert uncompiled KeymanWeb source scripts -->              
-    <script src="../source/kmwstring.js" type="application/javascript"></script>
-    <script src="../source/kmwbase.js" type="application/javascript"></script>
-    <script src="../source/keymanweb.js" type="application/javascript"></script>
-    <script src="../source/kmwosk.js" type="application/javascript"></script>
-    <script src="../source/kmwnative.js" type="application/javascript"></script>
-    <script src="../source/kmwcallback.js" type="application/javascript"></script>
-    <script src="../source/kmwkeymaps.js" type="application/javascript"></script>
-    <script src="../source/kmwlayout.js" type="application/javascript"></script>
-    <script src="../source/kmwinit.js" type="application/javascript"></script>             
+    <script src="../build/keymanweb.js" type="application/javascript"></script>           
```

Comment:
I thought *.ts gets compiled into  `../build/keymanweb.js`. Maybe the whole page `uncompiled.html` goes away?

Reply:
`../build/keymanweb.js` is the first-stage compiled JavaScript file - it's the direct output of the `tsc` command, before Google Closures performs the second stage.

As a result, no minification has been done upon `../build/keymanweb.js` - the 'fully compiled/minified' version is at `../output/keymanweb.js`.  The latter also has to deal with a second layer of source mapping, which may not be ideal when testing on all browsers.  The lack of minification on the former would prove helpful toward testing for browsers that don't recognize source mapping, so the majority of tests will use this intermediate product.